src/layouts/settings/Settings.tsx

```diff
@@ -173,7 +173,7 @@ export const Settings = () => {
         radius="md"
       >
         <Text size="sm" style={{ marginBottom: 10 }} weight={500}>
```

Comment:
can we use the `const { t } = useTranslation()` hook here?

Reply:
This is code from the legacy component. So this would be an optional fix from my point of view.

Reply:
Fixed on my machine. I'll commit to my branch today.